sections/summary.tex

```diff
@@ -0,0 +1,108 @@
+Without modification, \gls{pam} will find topics with the same structure as our taxonomy, but the topics will not actually be based on the taxonomy.
+To account for this, we lock taxonomy for the words in the observed documents to be in the corresponding topics within the pachinko \gls{dag} instead of continuously sampling them using Gibbs sampling.
+This creates a constant context for the taxonomy topics, which the documents with unobserved taxonomies will be fitted around.
+
+%evaluation
+The main metric used, in this experiment, is the topic coherence metric.
+This metric indicates how semantically similar the top words within each topic are, and will be an indication of the quality of the topics of a topic model.
+The second metric used in the experiment is topic difference.
+Topic difference is another metric that is used to check the quality of the topic model.
+It is based on the assumption that a good topic model will have little overlap between topics.
+
+Before fitting our models, we search for the optimal hyperparameters for our models, since these can vary based on the dataset.
+The hyperparameters we are testing in this grid search are the number of topics $K$ and an $\alpha$ and $\eta$ parameter.
+We only run the grid search on the standard \gls{lda} model, with the assumption that the number of topics that performs well for this model, also performs well for the metadata models, when the same dataset is used.
+Based on the topic coherence of the model, we choose $K = 90$, $\alpha = 0.01$, and $\eta = 0.1$ as the hyperparameters for all models in the experiment.
+
+For the results, the topic coherence of \gls{lda} is 0.520, author-topic is 0.335, category-topic is 0.370, and taxonomy-topic is 0.660.
+This shows that the Author and Category-Topic models performed the worst in topic coherence, whereas the Taxonomy-Topic model is outperforming all other models.
+However, the run time of the Taxonomy model is much higher than the standard \gls{lda}.
+
+%discussion
+In the analysis of the models, we explored the highest probable words from each model, except the taxonomy-topic model, that would appear in a chosen article.
+It is seen that there is a large amount of overlap between the models, but the words that best summarized the article came from the \gls{lda} model.
+
+For the author-topic and category-topic model, we we also able to analyze the similarity between author pairs and category pairs.
+This similarity is calculated with the symmetric Kullback-Leibler divergence of either the author-topic distributions or the category-topic distributions.
+This gives the intuition of how similar authors are based on the topics they write, and categories in the topics they cover.
+It is seen that it is difficult to find specifically why authors are similar, because they often write about many different subjects.
```

Comment:
```suggestion
It is seen that it is difficult to find specifically why authors are similar because they often write about many different subjects.
```